### PR TITLE
Utils::is_email: remove dependency on ilErr

### PR DESCRIPTION
**Pull request by klees**
This dependency seems to be outdated, as `ilErr` is not really used anywhere and always an element in the `DIC` and an object when ILIAS is initialized via `ilInitialisation`. It also was in the `DIC` and an object in the setup before #1071 was merged. So the second branch seems to be abandoned anyway and thus can be removed.

This should fix https://mantis.ilias.de/view.php?id=23683

### CODE REVIEW

**Review by chfsx — APPROVED** [review]
LGTM, but I'm not sure if I'm the person to merge that?

**Comment by alex40724** [comment]
Thanks. I think our mail maintainer @mjansenDatabay should have a final say.

**Comment by mjansenDatabay** [comment]
LGTM

**Pull request merged**
